src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/BackgroundService.cs

```diff
@@ -28,6 +28,7 @@ public abstract class BackgroundService : IHostedService, IDisposable
         /// the lifetime of the long running operation(s) being performed.
         /// </summary>
         /// <param name="stoppingToken">Triggered when <see cref="IHostedService.StopAsync(CancellationToken)"/> is called.</param>
+        /// See <related type="Article" href="/dotnet/core/extensions/workers">Worker Services in .NET</related> for implementation guidelines.
```

Comment:
This is incorrect, AFAIK. The `<related>` tag shouldn't have any surrounding text. If you want to have descriptive text like that, I think you need to use a `<see href="url">text</see>` within the summary or remarks.